### ISSUE TITLE
Symmetrize issue

### ISSUE THREAD

**Issue by djm87** [mtex-toolbox/mtex#584]
Hi guys, 

MTEX 5.4 Matlab 2020b 

I ran into a weird symmetrize issue where I wasn't getting the expected number twin components. The following code demonstrates the issue and to first order the crystal symmetries look the same: 

```matlab
%% Load from cif and define twins
cs1 = crystalSymmetry.load('Mg-Magnesium.cif')
sS1 = slipSystem.twinT1(cs1);

% consider all symmetrically equivlent slip systems
sS1 = sS1.symmetrise('antipodal')

%% Similarly define
mtexdata twins silent

sS2 = slipSystem.twinT1(ebsd{2}.CS);

% consider all symmetrically equivlent slip systems
sS2 = sS2.symmetrise('antipodal')
```
Outputs:
```
sS1 = slipSystem  
 mineral: Mg (6/mmm, X||a*, Y||b, Z||c)
 size: 6 x 1
  U    V    T    W  | H    K    I    L CRSS
  1   -1    0    1   -1    1    0    2    1
  1   -1    0   -1   -1    1    0   -2    1
  0    1   -1    1    0   -1    1    2    1
  0   -1    1    1    0    1   -1    2    1
  1    0   -1    1   -1    0    1    2    1
  1    0   -1   -1   -1    0    1   -2    1

sS2 = slipSystem  
 mineral: Magnesium (6/mmm, X||a*, Y||b, Z||c)
 size: 9 x 1
  U    V    T    W  | H    K    I    L CRSS
  1   -1    0    1    1   -1    0   -2    1
  1   -1    0   -1   -1    1    0   -2    1
  0   -1    1    1    0    1   -1    2    1
  1   -1    0    1   -1    1    0    2    1
  1   -1    0   -1    1   -1    0    2    1
  0    1   -1    1    0   -1    1    2    1
  0   -1    1    1    0   -1    1   -2    1
  1    0   -1    1   -1    0    1    2    1
  1    0   -1   -1   -1    0    1   -2    1
```
Regards, 
Dan

**Comment by ralfHielscher** [comment]
It seems this was already fixed in 2d89fb3e957d855ea6b9afbcc150dea7cb12da1b. Could you checkout the current development version? I will also release a new beta shortly.

Ralff.

**Comment by djm87** [comment]
That fixes the issue. 

Dan
